### DISCUSSION TITLE
tutorials/l3evpn/rt5-only/l3evpn/

### DISCUSSION THREAD

**Discussion by giscus** [srl-labs/learn-srlinux#163] [comments]
# tutorials/l3evpn/rt5-only/l3evpn/

Open documentation for open Network OS

https://learn.srlinux.dev/tutorials/l3evpn/rt5-only/l3evpn/

**Comment by isnazarov** [comment]
Hi there,
Thank you for your hard work!
`ip-vrf-1` - there is no such name in the config. Perhaps should use `tenant-1`?

**Reply by hellt** [reply]
thanks @isnazarov 
indeed, fixed the outdated netinst name in https://github.com/srl-labs/learn-srlinux/commit/2b18bbceb3ab7f7bcfadcdae1ed6fb7de5c93329

**Comment by isnazarov** [comment]
Could you please explain what the route ID is for (the 2nd column in the route-table output)?

**Reply by hellt** [reply]
it is an internal unique id assigned by the route owner 

https://yang.srlinux.dev/v24.10.1/tree?path=%2Fnetwork-instance%5Bname%3D*%5D%2Froute-table%2Fipv4-unicast%2Froute%5Bipv4-prefix%3D*%5D%5Broute-type%3D*%5D%5Broute-owner%3D*%5D%5Bid%3D*%5D%5Borigin-network-instance%3D*%5D%2Fid

**Comment by manumamm** [comment]
This is great, thanks a lot for this!
A small typo:
"checking that RT value is auto-derived from the EVI we set. And RD value is set manually to the same value on both leafs" => RD and RT have to be changed

**Reply by hellt** [reply]
thanks, fixed this
